internal/graph/model/apiextensions.go

```diff
@@ -171,7 +162,6 @@ func GetComposition(cmp *extv1.Composition) Composition {
 			},
 			WriteConnectionSecretsToNamespace: cmp.Spec.WriteConnectionSecretsToNamespace,
 		},
-		Status: GetCompositionStatus(cmp.Status),
```

Comment:
does this change the schema?

Reply:
Not sure. I did deploy these changes to prod and didn't see an issue 🤷